language/en-GB/com_media.ini

```
@@ -54,7 +54,7 @@ COM_MEDIA_FIELD_CHECK_MIME_DESC="Use MIME Magic or Fileinfo to try to verify fil
 COM_MEDIA_FIELD_CHECK_MIME_LABEL="Check MIME Types"
 COM_MEDIA_FIELD_IGNORED_EXTENSIONS_DESC="Ignored file extensions for MIME type checking and restricted uploads."
 COM_MEDIA_FIELD_IGNORED_EXTENSIONS_LABEL="Ignored Extensions"
-COM_MEDIA_FIELD_ILLEGAL_MIME_TYPES_DESC="A comma separated list of illegal MIME types to upload (blacklist)."
+COM_MEDIA_FIELD_ILLEGAL_MIME_TYPES_DESC="A comma separated list of illegal MIME types to upload."
```

Similar to administrator language file:
```suggestion
COM_MEDIA_FIELD_ILLEGAL_MIME_TYPES_DESC="A comma separated list of MIME types that can not be uploaded."
```